test/git-mock.js

```diff
@@ -26,6 +26,9 @@ if (cmd === 'rev-parse') {
   } else {
     console.log('Commit message');
   }
+} else if (cmd === 'log') {
+  console.log('foo bar baz');
+  process.exit(0);
```

Comment:
FWIW, process.exit is synchronous but writing to stdout/stderr is async, so calling process.exit can truncate output. If possible, it is usually better to set process.exitCode and allow the program to naturally exit.

Reply:
ah, good to know!

Reply:
Yeah, so not really a big deal here, but might be worth doing it differently anyway since it should be pretty straightforward.

Here's the docs that mention this BTW: https://nodejs.org/api/process.html#process_process_exit_code